src/rastervision/crs_transformers/rasterio_crs_transformer.py

```diff
@@ -4,22 +4,48 @@
 
 
 class RasterioCRSTransformer(CRSTransformer):
+    """Transformer for a RasterioRasterSource.
+
+    This assumes that the map coordinates are always in lon/lat format.
```

Comment:
Is there a reason to not allow for a CRS to be optionally passed in to __init__, defaulting to EPSG:4326?

Reply:
I added this.